### ISSUE TITLE
can i get every sentence probabality

### ISSUE THREAD

**Issue by yqtaowhu** [tensorflow/nmt#65]
Hello friends:
when i want to get every sentence probabality,how i can change the code,
where `logits, sample_id, final_context_state
`i calculate softmax(logits)?
hope you can reply!!!

**Comment by yqtaowhu** [comment]
i mean  in this code in model.py` infer_logits, infer_summary, _, sample_words = self.infer(sess)`
if i want get every word probabality , should i softmax(infer_logits)??
